### PR TITLE
Importable

### PR DESCRIPTION
**Pull request by pp-mo**
Moved source + fixed imports make a barely importable, functional module.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.installed.cfg
+
+# Installer logs
+pip-log.txt
+
+#shared objects
+*.so
+
+#config
+cf_units/etc/site.cfg
```
Comment (pelson) [inline .gitignore]: Probably not necessary 😉

Comment (pp-mo) [inline .gitignore]: Ooh I see.
I just copied the cf_utils one.
I hadn't realised that would show up so clearly !

File: iris_grib/__init__.py [hunk 2]
```diff
@@ -1 +1,1098 @@
+# (C) British Crown Copyright 2010 - 2016, Met Office
+#
+# This file is part of Iris.
```
Comment (pelson) [inline iris_grib/__init__.py]: Not yet, but we will need to decide on this (and whether we need a header at all).

File: .gitignore (hunk 1 above)
Comment (rhattersley) [inline .gitignore]: I'd like to test the doc building ... so please can we fix this in a follow-up PR.

**Pull request merged**